### ISSUE TITLE
NoSuchMethodError is thrown with realm 0.82.1

### ISSUE THREAD

**Issue by njovy** [uPhyca/stetho-realm#6]
When I tried to open a realm file on a chrome inspector, it crashes.

```
 java.lang.NoSuchMethodError: No direct method <init>(Ljava/lang/String;Z[B)V in class Lio/realm/internal/SharedGroup; or its super classes (declaration of 'io.realm.internal.SharedGroup' appears in /data/app/com.oldroid.apps.votox.internal.dev-3/base.apk)
            at com.uphyca.stetho_realm.RealmPeerManager.openSharedGroupForImplicitTransactions(RealmPeerManager.java:130)
            at com.uphyca.stetho_realm.RealmPeerManager.getDatabaseTableNames(RealmPeerManager.java:52)
            at com.uphyca.stetho_realm.Database.getDatabaseTableNames(Database.java:90)
            at java.lang.reflect.Method.invoke(Native Method)
            at java.lang.reflect.Method.invoke(Method.java:372)
            at com.facebook.stetho.inspector.MethodDispatcher$MethodDispatchHelper.invoke(MethodDispatcher.java:96)
            at com.facebook.stetho.inspector.MethodDispatcher.dispatch(MethodDispatcher.java:67)
            at com.facebook.stetho.inspector.ChromeDevtoolsServer.handleRemoteRequest(ChromeDevtoolsServer.java:129)
            at com.facebook.stetho.inspector.ChromeDevtoolsServer.handleRemoteMessage(ChromeDevtoolsServer.java:111)
            at com.facebook.stetho.inspector.ChromeDevtoolsServer.onMessage(ChromeDevtoolsServer.java:87)
            at com.facebook.stetho.websocket.WebSocketSession$1.handleTextFrame(WebSocketSession.java:176)
            at com.facebook.stetho.websocket.WebSocketSession$1.onCompleteFrame(WebSocketSession.java:136)
            at com.facebook.stetho.websocket.ReadHandler.readLoop(ReadHandler.java:46)
            at com.facebook.stetho.websocket.WebSocketSession.handle(WebSocketSession.java:45)
            at com.facebook.stetho.websocket.WebSocketHandler.doUpgrade(WebSocketHandler.java:154)
            at com.facebook.stetho.websocket.WebSocketHandler.handleSecured(WebSocketHandler.java:106)
            at com.facebook.stetho.server.SecureHttpRequestHandler.handle(SecureHttpRequestHandler.java:52)
            at org.apache.http.protocol.HttpService.doService(HttpService.java:248)
            at org.apache.http.protocol.HttpService.handleRequest(HttpService.java:192)
            at com.facebook.stetho.server.LocalSocketHttpServer$WorkerThread.run(LocalSocketHttpServer.java:233)
```

**Comment by zaki50** [comment]
@njovy thanks! I'll fix it soon.

**Comment by zaki50** [comment]
@njovy we released 0.7.0 which supports realm-java 0.82.*.

Please reopen this issue if you find some problems.

thanks.

**Comment by njovy** [comment]
I tested 0.7.0 with 0.82.1 and it worked like a charm.
Thank you for this awesome library!
